Cannot init a new Remix project with NPM 8

Using `npm init remix` with NPM v8.0.0 is throwing an error. The command executes successfully, but after entering the path where I want the project created the following error is shown:

```
npm < version 6 is not supported (you're using 8.0.0)
npm ERR! code 1
npm ERR! path xxx
npm ERR! command failed
npm ERR! command sh -c create-remix

npm ERR! A complete log of this run can be found in:
npm ERR!     xxx
```

As a result, no subsequent questions are asked and the project skeleton is not created.

--- a/packages/create-remix/cli.ts
+++ b/packages/create-remix/cli.ts
@@ -140,7 +140,7 @@ function getNpxCommand(script: string): string {
   let version = execSync("npm --version").toString().trim();
   let major = semver.major(version);
 
-  if (major === 7) {
+  if (major >= 7) {
     return `npx --yes --quiet ${script}`;
   } else if (major === 6) {
     return `npx --quiet ${script}`;

